### ISSUE TITLE
extend input possible?

### ISSUE THREAD

**Issue by mheidt** [graphql-java-generator/graphql-maven-plugin-project#136]
Hi Etienne,

I am using multiple graphql files and hence used this a lot:
```
extend type Query {
    foo:bar
}
```

Now i would like to extend inputs as well.
But unfortunately this doesn't work.
The extended fields are missing in the generated code.

Does the spec say, that it should be possible? The code generator does't fail, if I have a construct like:
```
enum ApplicationFilterEnum {
    NAME
    SHORTNAME
    APPLICATION_TYPE_NAME
}

extend input ApplicationFilterField {
    name: ApplicationFilterEnum
}
```

I am using 1.18.3

**Comment by mheidt** [comment]
And while you are at it:
extend enums is overwriting and not extending.

**Comment by etienne-sf** [comment]
I'll add tests for all possible ´extend´ use. 
And I fear, like you stated, that some corrections will need to be shipped with...

**Comment by etienne-sf** [comment]
This is now available in the 1.18.7 release
